src/index.js

```diff
@@ -0,0 +1,14 @@
+const createThemeProvider = require('./create-theme-provider');
+const createWithTheme = require('./create-with-theme');
+const channel = require('./channel');
+
+module.exports = {
+  channel,
+  withTheme: createWithTheme(),
+  ThemeProvider: createThemeProvider(),
+  createTheming: (customChannel = channel) => ({
```

Comment:
if other packages do have they own create function, then you would end up with renaming all of them:

```js
import { create as createJss } from 'jss';
import { create as createInjectSheet } from 'react-jss';
import { create as createTheming } from 'theming';
```

vs

```js
import { createJss } from 'jss';
import { createInjectSheet } from 'react-jss';
import { createTheming } from 'theming';
```